### PR TITLE
Split off request collector

### PR DESCRIPTION
**Pull request by karlb**
as planned in https://github.com/raiden-network/raiden-monitoring-service/issues/60.

### CODE REVIEW

File: monitoring_service/tests/conftest.py [hunk 1]
```diff
@@ -12,4 +12,4 @@
 from raiden_libs.test.fixtures.address import *  # noqa
 from raiden_libs.test.fixtures.client import *  # noqa
 from raiden_contracts.tests.fixtures import *  # noqa
-from monitoring_service.test.fixtures import *  # noqa
+from monitoring_service.tests.fixtures import *  # noqa
```
Comment (palango) [inline monitoring_service/tests/conftest.py]: You need to change the ignore path in the `.coveragerc` file as well

File: request_collector/cli.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import click
+from eth_utils import is_checksum_address
+
+from request_collector.server import RequestCollector
+from monitoring_service.state_db import StateDBSqlite
+from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
+from raiden_libs.transport import MatrixTransport
+
+log = logging.getLogger(__name__)
+contract_manager = ContractManager(contracts_precompiled_path())
```
Comment (palango) [inline request_collector/cli.py]: Is this used?

File: request_collector/cli.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+DEFAULT_REQUIRED_CONFIRMATIONS = 8
```
Comment (palango) [inline request_collector/cli.py]: this seems unused as well.

**Comment by palango** [comment]
And the commands in the `Makefile` need some fixing so they run on `./request_collector` as well.

**Comment by karlb** [comment]
All good points. Addressed in ebd03a0.

**Pull request merged**